common/index.ts

```diff
@@ -38,6 +38,19 @@ export const OPENID_AUTH_LOGOUT = '/auth/openid/logout';
 export const SAML_AUTH_LOGOUT = '/auth/saml/logout';
 export const ANONYMOUS_AUTH_LOGOUT = '/auth/anonymous/logout';
 
+export const ANONYMOUS_ROUTES = [
```

Comment:
Can these endpoints be configured in such a way not to call `isReadOnly` at all? I'd prefer not to maintain a list and instead configure this where the route is declared.

Reply:
This was largely addressed with `isAnonymousPage(...)` in a future iteration